src/main/java/net/neoforged/neoforge/common/world/ModifiableStructureInfo.java

```diff
@@ -66,7 +65,7 @@ public StructureInfo getModifiedStructureInfo() {
     @ApiStatus.Internal
     public void applyStructureModifiers(final Holder<Structure> structure, final List<StructureModifier> structureModifiers) {
         if (this.modifiedStructureInfo != null)
-            throw new IllegalStateException(String.format(Locale.ENGLISH, "Structure %s already modified", structure));
+            return;
```

Comment:
Adjust the javadoc of this method to match how this method now operates. (It no longer throws an exception, and the biome is only modified on the first invocation of this method.)